kernels/simple_mult/gendata.py

```diff
@@ -0,0 +1,44 @@
+# simple script to generate inputs and expected outputs for simple_mult
+import numpy as np
+from numpy import typing as npt
+from typing import Dict
+
+def create_header(file_name: str, size: int, variables: Dict[str, npt.NDArray]) -> None:
+    with open(file_name, "w") as f:
+        includes = ["#include <stdint.h>",
+                    "#pragma once", 
+                    "",
+                    f"#define N {size}",
+                    ""]
+        includes = "\n".join(includes)
+        variable_names = list(variables.keys())
+        variables_string = [f"extern const int32_t {i}[N];" for i in variable_names]
+        variables_string = "\n".join(variables_string)
+        f.write(includes)
+        f.write(variables_string)
+        f.write("\n")
+
+def create_data(file_name: str, size: int, variables: Dict[str, npt.NDArray]):
+    includes = ['#include "data.h"', "", ""]
+    includes = "\n".join(includes)
+    with open(file_name, "w") as f:
+        f.write(includes)
+        for variable_name, variable_value in variables.items():
+            f.write(f"const int32_t {variable_name}[N] = "+"{\n")
+            variable_str = ["\t"+str(i) for i in variable_value]
+            f.write(",\n".join(variable_str))
+            f.write("\n};\n\n")
```

Comment:
These look like they can go into some `gen_data_util.py` function 🤖

Reply:
Definitely! Something for later :wink: